app/controllers/users_controller.rb

```diff
@@ -1,8 +1,16 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only:[:index, :edit, :update, :destroy]
```

Comment:
use except: instead of only in this case because expect has less action than only